src/os/bluestore/BlueStore.cc

```diff
@@ -5050,33 +5050,33 @@ void BlueStore::_init_logger()
                     "Read EIO errors propagated to high level callers");
   b.add_u64_counter(l_bluestore_reads_with_retries, "bluestore_reads_with_retries",
                     "Read operations that required at least one retry due to failed checksum validation",
-		    "rw_r", PerfCountersBuilder::PRIO_USEFUL);
+		    "fail", PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
why fail?

Reply:
The reasoning behind this decision was that `bluestore_reads_with_retries` is the only BlueStore counter that indicates failure. But maybe we should go with a more conventional nick-- @aclamk do you have anythoughts?

Reply:
what's about rd_r?